### PR TITLE
A few small fixes

### PR DESCRIPTION
**Pull request by beeeees**
hey @kynatro i just cloned the repo and found a few small bugs that kept it from working. check out the code to see the changes, here's the summary. can you review for me?

- defining some missing variables referenced in the css helpers, $speed and $easing
- adding babel plugin to dev dependencies (looks like this is required by bootstrap4 but was not included, not sure if an error they are resolving or not)
- removing trailing ) in gruntfile

### CODE REVIEW

**Comment by mrkantz** [comment]
@beeeees Good call! I actually ran into these as well with the Phantom stuff.

File: package.json [hunk 1]
```diff
@@ -28,6 +28,7 @@
   },
   "dependencies": {
     "autoprefixer": "^6.0.3",
+    "babel-plugin-transform-es2015-modules-strip": "^0.1.0",
```
Comment (kynatro) [inline package.json]: What does this package actually do and why was it added?

**Comment by mrkantz** [comment]
@kynatro Should we also add the Procfile and Rakefile to the boilerplate? Just noticed those when I was getting this project setup.

**Comment by beeeees** [comment]
@kynatro it's referenced in the Bootstrap JS .babelrc file as a plugin but wasn't installed in the dependencies at any point so it was throwing an error when running middleman. 

https://www.dropbox.com/s/kmhlqg8dmrcxy80/Screenshot%202017-01-06%2014.40.40.png?dl=0

It says it's purpose is to "A Babel plugin that strips import and export declarations.". I dont know much about babel but i also couldn't find anyone reporting it as a BS bug.  However, installing it fixes the issue.

**Comment by kynatro** [comment]
Ahhh, that was added in the version of Bootstrap I updated to after importing changes from Mindflash's codebase.

Blah, that's lame. Hopefully it will go away in a later version. There shouldn't be anything in `devDependencies` that are required to make the project build for production operation.

**Pull request merged**